Dockerfile

```diff
@@ -28,7 +31,9 @@ USER mailhog
 
 WORKDIR /home/mailhog
 
-ENTRYPOINT ["MailHog"]
```

Comment:
Should probably be set as `CMD ["Mailhog"]` now, then it'll act as the default argument for the entrypoint; that allows overriding the default command to, for example, get a shell in the container.

You can draw some inspiration from the official images, for example https://github.com/docker-library/mysql/blob/333935aa6612376d58737a8cab0e3f5df370585a/8.0/docker-entrypoint.sh

Reply:
Sure, but I am adding only the feature to turn on basic auth with envs. Anyway this can be useful for MailHog's author :+1:

Reply:
The `$@` in `exec "MailHog $@"` will allow to keep current flow now.

I'm not going to change @ian-kent's flow, just adding an improvement.